AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/gcloud/GoogleCloudApi.kt

```diff
@@ -138,6 +147,13 @@ private class GoogleCloudApiImpl(
         GcsPath.create(blob)
     }
 
+    override suspend fun downloadAdditionalApk(
```

Comment:
i'm not sure if this work should be done by CI action.

I think this should happen on the androidx'dev side.